app/scripts/src/content/content-controller.js

```diff
@@ -31,8 +31,8 @@ ContentController.prototype = {
   },
 
   onSearchError: function(status, response) {
-    this.sendAnalyticsEvent({ name: 'onSearchError', value: status });
-    console.error('traktflix: Search error', status, response);
+    this.sendAnalyticsEvent({ name: 'onSearchError', value: status + (this.item && this.item.title) ? this.item.title : "Invalid item title" });
```

Comment:
I do agree with you, but I thought better safe than sorry. It is better to have a scenario that will never trigger than to have an error in the console which you'll never be able to catch.
Having  said that I don't like having code that serves no purpose, so I would still suggest to remove it in the future.